homeassistant/components/todoist/calendar.py

```diff
@@ -231,10 +231,7 @@ def _parse_due_date(data: dict) -> datetime:
     # Add time information to date only strings.
     if len(data["date"]) == 10:
         data["date"] += "T00:00:00"
-    # If there is no timezone provided, use UTC.
-    if data["timezone"] is None:
-        data["date"] += "Z"
-    return dt.parse_datetime(data["date"])
+    return dt.as_utc(dt.parse_datetime(data["date"]))
```

Comment:
You're now assuming that the todoist local timezone is the same as the HA instance timezone. This doesn't have to be. Is there another date field that is UTC?